### ISSUE TITLE
[InteractiveSegmentation] Progress bar should reset after running and unloading data

### ISSUE THREAD

**Issue by ashishsingh18** [CBICA/CaPTk2.0#142]
![image](https://user-images.githubusercontent.com/43445206/76454264-bde96080-63aa-11ea-8061-73207eb59523.png)

**Comment by ashishsingh18** [comment]
This might be the case with other plugins as well...

**Comment by dboun** [comment]
You're right. It generally should reset when both the following are true at the same time:

- There is not a running calculation
- The set of the loaded non-label images change (either new images are added, or existing ones are removed)

I think this can be done in
```cpp
  // reimplemented from QmitkAbstractView
  void NodeAdded(const mitk::DataNode* node) override;

  // reimplemented from QmitkAbstractView
  void NodeRemoved(const mitk::DataNode* node) override;
```

which are part of QmitkAbstractView or mitk::ILifecycleAwarePart
